### PR TITLE
Added Snackbar qtquick object

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>
# 🎉 New feature

Related with this issue https://github.com/ignitionrobotics/ign-gui/issues/44

## Summary

Create a IgnSnackBar QtQuick object to show message to the user in the same manner by plugins

It could be :
 - persistant and then you need to click inside the GUI
 - or with a timeout

snackbar

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: include/ignition/gui/resources.qrc [hunk 1]
```diff
@@ -9,6 +9,7 @@
   <file>qml/IgnRulers.qml</file>
   <file>qml/IgnSortFilterModel.qml</file>
   <file>qml/IgnSpinBox.qml</file>
+  <file>qml/IgnSnackBar.qml</file>
```
Comment (chapulina) [inline include/ignition/gui/resources.qrc:12]: alphabetic order

File: include/ignition/gui/qml/IgnSnackBar.qml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  id: snackbar
+  modal: true
+  focus: true
+  x: (window.width - width) / 2
+  y: window.height - window.height / 6
+  width: window.width - window.width / 6
+  contentHeight: notificationColumn.height
+
+  property int duration: 1000
+  property bool persistant: false
```
Comment (chapulina) [inline include/ignition/gui/qml/IgnSnackBar.qml:34]: Since these are part of the component's public API, it would be good to document them

**Review by chapulina — COMMENTED** [review]
Works well for me! I think this can go into Citadel, right?

It would be good to adopt it in as many places as we can find, so we encourage other developers to follow the same pattern.

**Comment by ahcorde** [comment]
superseed by https://github.com/ignitionrobotics/ign-gui/pull/369

**Pull request closed without merging**